paimon-common/src/main/java/org/apache/paimon/utils/RoaringBitmap32.java

```diff
@@ -39,6 +40,10 @@ public RoaringBitmap32() {
         this.roaringBitmap = new RoaringBitmap();
     }
 
+    public RoaringBitmap32(RoaringBitmap roaringBitmap) {
```

Comment:
We should not support this, `RoaringBitmap` should be hided.

Reply:
okay, can I just change it to private?